object_database/web/cells/cells.py

```diff
@@ -2016,6 +2122,9 @@ def sortsAs(self):
 
 
 class Grid(Cell):
+    # TODO: Do the individual data cells (in grid terms) need to be actual Cell objects?
+    # Is there a way to let the Components on the front end handle the updating of the
+    # data that gets presented, without having to wrap each datum in a Cell object?
```

Comment:
Yes it needs to be a cell, because we almost always render a 'Subscribed' in a grid so that the values can tick. I can imagine an optimization where we would detect if the rendererFun returned a python primitive like text or something, in which case we could dispense with sending a cell.